BLSQ/iaso
# IA 1604 / IA 1605: improve orgunits web UI

Explain what problem this PR is resolving

- `OrgUnitCreationDetails` was shown even when creating a new OrgUnit but the values are only readable and not editable so there was no reason to show them


## Self proof reading checklist

## Changes

- moved `OrgUnitCreationDetails` from orgUnit creation and show it only on orgUnit detail
- refactored `OrgUnitCreationDetails` and split it into a separate component. Use a Table with rows instead of a list of `InputComponent` as those info are not editable
- some layout update to make sure layout don't break


## How to test

Go to Orgunit > Create

![Capture d’écran du 2022-12-19 11-45-44](https://user-images.githubusercontent.com/25134301/208408957-42ba8ead-3f63-4a04-a2dd-570241c415e4.png)


Go to OrgUnit detail 

If an instance is linked to this orgunit, you should see the form. 
![Capture d’écran du 2022-12-19 11-48-06](https://user-images.githubusercontent.com/25134301/208409133-933be002-b13d-4b79-95b6-ae069c38be21.png)


If not, you should only see the same fields when you create an orgunit 
![Capture d’écran du 2022-12-19 11-48-06](https://user-images.githubusercontent.com/25134301/208409194-ed527d23-80cc-4926-8634-5a6dc7b25721.png)


## Print screen / video

[recording-2022-12-19-11-44-21.webm](https://user-images.githubusercontent.com/25134301/208409258-261d8285-35b0-436a-b7f4-1df5c914a4b1.webm)
[recording-2022-12-19-11-40-09.webm](https://user-images.githubusercontent.com/25134301/208409264-ec2bfb35-1f71-452f-a2b9-975480ae2d9c.webm)
